activation/nipost.go

```diff
@@ -388,23 +366,74 @@ func (nb *NIPostBuilder) submitPoetChallenge(
 	})
 }
 
-// Submit the challenge to all registered PoETs.
+// Submit the challenge to registered PoETs with given addresses.
```

Comment:
The addresses argument was removed already :).
```suggestion
// Submit the challenge to registered PoETs.
```